src/Web/Microsoft.NET.Sdk.Web.Targets/Sdk.props

```diff
@@ -14,11 +14,8 @@ Copyright (c) .NET Foundation. All rights reserved.
   <!-- Default properties that shouldn't be replaced by the microsoft.net.sdk.props -->
   <PropertyGroup>
     <DebugSymbols Condition="'$(DebugSymbols)' == ''">true</DebugSymbols>
-    <PreserveCompilationContext Condition="'$(PreserveCompilationContext)' == ''">true</PreserveCompilationContext>
   </PropertyGroup>
 
-  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
```

Comment:
We'll move this declaration in to the Razor.Sdk. This should allow a user to just reference the Razor.Sdk in a class library and have it build.

Reply:
Why remove it? I would expect WebSdk to have reference to all its required Sdks. Even if Razor Sdk has another one I would think it to no-op since all it does is add an import. MSBuild should handle duplicate imports without any issues.

Reply:
Having it both in WebSdk and RazorSdk results in warnings: `warning MSB4011: "\.dotnet\x64\sdk\2.1.300-preview2-008173\Sdks\Microsoft.NET.Sdk\Sdk\Sdk.props" cannot be imported again`